modules/web/src/app/settings/admin/interface/template.html

```diff
@@ -82,8 +82,22 @@
         </div>
         <mat-checkbox [(ngModel)]="settings.enableDashboard"
                       (change)="onSettingsChange()"
-                      id="km-enable-kubernetes-dashboard-setting"></mat-checkbox>
-        <km-spinner-with-confirmation [isSaved]="isEqual(settings.enableDashboard, apiSettings.enableDashboard)"></km-spinner-with-confirmation>
+                      [disabled]="!isKubernetesDashboardFeatureGatesEnabled()"
+                      id="km-enable-kubernetes-dashboard-setting">
+          <ng-container *ngIf="!isKubernetesDashboardFeatureGatesEnabled()">
+
+            <mat-hint *ngIf="!isOIDCKubeCfgEndpointEnabled">This feature is disabled. Visit the
+              <a href="https://docs.kubermatic.com/kubermatic/main/tutorials-howtos/oidc-provider-configuration/share-clusters-via-delegated-oidc-authentication/"
```

Comment:
FYI i have linked the same link and docs has something related to following Feature gates as per discussion with patryk docs will be extended for same url.

<img width="748" alt="configuration" src="https://user-images.githubusercontent.com/17727069/202126282-7f8ebd56-504f-46f6-99c4-fff21cd69eac.png">